Source/Resources/Content/AadIam/UserSettings/Configuration.json

```diff
@@ -1,4 +1,5 @@
 {
+  "description":"What is it: Additional controls for what users can do in Azure, including: registering new applications, accessing the portal, and using LinkedIn connectors. \n Simeon Notes: Simeon's baseline allows users to register new applications and access the Azure portal, but does not allow LinkedIn connectors. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/users-default-permissions#to-restrict-the-default-permissions-for-member-users",
```

Comment:
**What does this do?**
Enables additional controls for users in Azure. The baseline allows users to register new applications and access the Azure portal, but does not allow LinkedIn connectors.

**Why should you use this?**
If you want your users to be able to register new applications and access the Azure portal, but not share data with LinkedIn to be more secure.

**What is the end user impact?**
Users may register new applications and access the Azure portal, but not access their LinkedIn connections within Microsoft apps. 

**Learn more**
[Default user permissions](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/users-default-permissions#to-restrict-the-default-permissions-for-member-users)